solutions/task_4.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+"""
+TASK: Implement classes "Temperature", "Person" using Python descriptor protocols to execute unittests without errors.
+"""
+import re
+import unittest
+from datetime import datetime
+
+
+class Temperature(object):
+
+    def __init__(self, value=0):
+        self.fahrenheit = value
+
+    def get_celsius(self):
```

Comment:
Use underscore to show user this these methods are internal. Use may decide that he has to use this methods, not ```celsius``` attr.
```def _get_celsius```
```def _set_celsius```

Reply:
Done